src/styles/brackets_codemirror_override.less

```diff
@@ -27,6 +27,7 @@
 
     > * {
         text-indent: 0px;
+        margin-left: -4px;
```

Comment:
@petetnt Not sure how to fix it in the hack itself, but I found out that just removing `padding: 0 @code-padding 0 0;` on line 26 also fixes the issue.